src/components/Nav.js

```diff
@@ -8,18 +8,24 @@ import ProfileAvatar from "../styles/ProfileAvatar";
 import AdminStatsBar from "./AdminStatsBar";
 import Hotkeys from "react-hot-keys";
 import {useHistory, Link} from "react-router-dom";
+import styled from "styled-components";
 
 function LeftSide({isLoggedIn, user, handleLogIn, handleLogOut}) {
+  const Logo = styled.img`
+  display: "inline-block";
+  vertical-lign: "middle"; 
+  margin-bottom: 6;
+  `;
```

Comment:
@JapneetSingh5 I suggest you to move here https://github.com/open-sauced/open-sauced/blob/master/src/styles/Grid/Float.js#L9-L18

Reply:
I move the Logo as is there and import it into Nav.js right?

Reply:
That good idea @JapneetSingh5 👍 proceed it!

Reply:
Done that 🥳